### PR TITLE
Create find azure global admin

### PR DESCRIPTION
**Pull request by ukpdb111**
New document explaining process to identify Global admins

### CODE REVIEW

File: articles/azure/azs-identify-global-admins.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: How to identify your Global Admins| UKCloud Ltd
+description: How to identify your Azure Global Admins for Microsoft Azure
+services: azure-stack
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: change to azure

File: articles/azure/azs-identify-global-admins.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+author: Paul Brown
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Identify your Global Admins
+toc_fullpath: Users/How To/azs-identify-global-admins.md
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: change to azs-how-identify-global-admins.md per guidelines (https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html#article-file-naming-conventions)

File: articles/azure/azs-identify-global-admins.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+toc_mdlink: azs-identify-global-admins.md
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: as above (and remember to change the actual file name!)

File: articles/azure/azs-identify-global-admins.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+title: How to identify your Global Admins| UKCloud Ltd
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: global administrators (unless there's a specific reason for it to be Global Admins)

File: articles/azure/azs-identify-global-admins.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+description: How to identify your Azure Global Admins for Microsoft Azure
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: global administrators

File: articles/azure/azs-identify-global-admins.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+services: azure-stack
+author: Paul Brown
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Identify your Global Admins
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: global admins (fine to use the abbreviation of admins in the toc title to keep it shorter)

File: articles/azure/azs-identify-global-admins.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+toc_fullpath: Users/How To/azs-identify-global-admins.md
+toc_mdlink: azs-identify-global-admins.md
+---
+
+# How to identify your Microsoft Azure Global Admins
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: global administrators

File: articles/azure/azs-identify-global-admins.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+## Overview
+
+The global administrator has access to all administrative features. By default, the person who signs up for an Azure subscription is assigned the global administrator role for the directory. Only global administrators can assign other administrator roles.
+
+### Intended audience
+
+To complete the steps in this guide, you must have appropriate access to your Microsoft Azure Portal.
+
+
+**Indentifying the Global administrators**
```
Comment (suehighmoor) [inline articles/azure/azs-identify-global-admins.md]: Make this a level two (##) heading

File: articles/azure/azs-identify-global-admins.md (hunk 1 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 2 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 3 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 4 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 5 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 7 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

File: articles/azure/azs-identify-global-admins.md (hunk 8 above)
Comment (ukpdb111) [reply articles/azure/azs-identify-global-admins.md]: done

**Comment by ukpdb111** [comment]
All changes made, new PR sent

**Comment by suehighmoor** [comment]
@cblackuk can you just confirm that the change to the metadata is correct (services: azure not azure-stack) as they are your docs?

**Comment by cblackuk** [comment]
@suehighmoor I would rather have it all azure-stack same as MS Docs. Can we change them all to reflect that please? @ukpdb111

**Comment by cblackuk** [comment]
As discussed with @suehighmoor  before merging azure needs to be azure-stack @ukpdb111

**Comment by cblackuk** [comment]
Duplicate PR

**Pull request closed without merging**